README.md

```diff
@@ -18,9 +18,9 @@ Specifically, in addition to the standard throughput tests, **sockperf** does th
 
    * GNU make tools: automake 1.7+, autoconf 2.57+, m4 1.4+ and libtool 1.4+
 
-   * A Compiler, among those tested are:
+   * A C++11 Compiler, among those tested are:
     
-     * . gcc4 (Ubuntu 9)
+     * gcc 4.8.5, 10.2.1 (Red Hat Enterprise Linux 7)
```

Comment:
Please consider removing gcc version and stay just with c++11 requirement.
Note: in fact ci checks following https://github.com/Mellanox/sockperf/blob/sockperf_v2/contrib/jenkins_tests/compiler.sh#L13

Reply:
I think that it is better to list checked compiler versions since some older gcc versions (like 4.8.5) declares support for C++11 but have some features not implemented or broken. In fact there are no compilers that supports [all of C++11 features](https://en.cppreference.com/w/cpp/11#Compiler_support).

Reply:
I would like to simplify this note because it is updated rarely.

Reply:
replaced with compiler list